### ISSUE TITLE
Mac 10.5.8

### ISSUE THREAD

**Issue by NostalgiaSaur** [jorio/Nanosaur#5]
Hello- 

First, I'm a huge fan of your port- I used to play this game all the time when I was a kid and it's kind of crazy to see it coming up again. I was hoping you'd be able to help me get it to work on Mac OS X Version 10.5.8.

I've been mainly working this on my laptop (10.12.6) and the old 10.5.8 computer. I started out trying to get your Nanosaur port on the 10.5.8 computer but couldn't because the old safari was having issues connecting to GitHub. So I got on my laptop and downloaded v1.4.0. It worked fine there, I could play the game with no problems. Then, using a flashdrive, I tried to move it over to the older computer, alternately trying to move the entire .dmg file and just the actual Nanosaur application itself. Each time, the Nanosaur application (opened directly from the flashdrive or moved onto the computer) showed up with no icon on the desktop (so it was just 'Nanosaur' underneath empty space) and would quit unexpectedly as soon as I tried to open it.

Essentially, I'm just wondering if you know what the issue could be, or if there's an older version of your port that would work on 10.5.8 and you'd be willing to let me use.

Sorry to bog you down with text, and thanks for your time, and for making this game playable again.

**Comment by jorio** [comment]
Hi!

Unfortunately, the oldest version of macOS that this port will work on reliably is 10.11. Porting Nanosaur “back” to older OS X releases would be quite the undertaking — Leopard is 13 years old at this point.

I don’t have an older version that’d work on 10.5 either, because I started the porting project this year. I get where you’re coming from, version “1.4.0” might suggest there are older versions, but it’s actually the first release I made — I just kept Pangea’s original versioning scheme (the last version of the game for OS 9 was 1.3.4, I just bumped that up to 1.4.0 for my release).

Sorry!
